website/dbt-versions.js

```diff
@@ -17,4 +21,9 @@ exports.versions = [
   }
 ]
 
-exports.versionedPages = []
+exports.versionedPages = [
+  {
+    "page": "docs/contributing/testing-a-new-adapter",
+    "firstVersion": "1.1",
+  }
+]
```

Comment:
This didn't accomplish quite what I'd hoped it would. I was expecting a banner on the page when an older version is selected, and I'm not seeing that

Reply:
jk it's working now!